tests/backup/backup_restore_basic_test.go

```diff
@@ -500,146 +500,6 @@ var _ = Describe("{DeleteAllBackupObjects}", func() {
 	})
 })
 
-// This testcase verifies schedule backup creation with a single namespace.
```

Comment:
Why removing this test case?

Reply:
It was decided in our test case scrub meeting that this is already being covered in many other tests and hence it is redundant.